tests/test_lateral_ccsd(t)_ene.py

```diff
@@ -0,0 +1,438 @@
+import pytest
+
+import os
+import sys
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from .utils import *
+from .addons import *
+
+import qcdb
+import qcengine as qcng
+
+@pytest.fixture
+def h2o():
+    return """
+    O
+    H 1 R
+    H 1 R 2 A
+
+    R=0.958
+    A=104.5
+"""
+
+@pytest.fixture
+def nh2():
+    return """
+    0 2
+    N
+    H 1 R
+    H 1 R 2 A
+    
+    R=1.008
+    A=105.0
+"""
+
+
+@pytest.mark.parametrize('mtd,opts', [
+    pytest.param('c4-ccsd(t)', {'cfour_basis': 'qz2p', 'cfour_scf_conv': 12, 'cfour_cc_conv': 12}, marks=using_cfour),
+    pytest.param('c4-ccsd(t)', {'basis': 'cfour-qz2p', 'cfour_scf_conv': 12, 'cfour_cc_conv': 12}, marks=using_cfour),
+    pytest.param('gms-ccsd(t)', {'basis': 'cfour-qz2p', 'gamess_ccinp__ncore': 0}, marks=using_gamess),
+    pytest.param('nwc-ccsd(t)', {'basis': 'cfour-qz2p', 'qc_module': 'tce'}, marks=using_nwchem),
+    pytest.param('nwc-ccsd(t)', {'basis': 'cfour-qz2p'}, marks=using_nwchem),
+    pytest.param('p4-ccsd(t)', {'basis': 'cfour-qz2p'}, marks=using_psi4),
+])
+def test_sp_ccsd_t_rhf_ae(mtd, opts, h2o):
+    """cfour/???/input.dat
+    #! single point CCSD(T)/qz2p on water
+
+    """
+    h2o = qcdb.set_molecule(h2o)
+    qcdb.set_options(opts)
+
+    e, jrec = qcdb.energy(mtd, return_wfn=True, molecule=h2o)
+
+    # from cfour
+    scf_tot = -76.0627484601
+    mp2_tot = -76.3329401286
+    ccsd_tot = -76.3384539547
+    ccsd_t_tot = -76.3457175511
+
+    mp2_corl = mp2_tot - scf_tot
+    ccsd_corl = ccsd_tot - scf_tot
+    ccsd_t_corl = ccsd_t_tot - scf_tot
+    t_corl = ccsd_t_tot - ccsd_tot
+
+    atol = 1.e-6
+
+    # cc terms
+    assert compare_values(ccsd_t_tot, e, tnm() + ' Returned', atol=atol)
+    assert compare_values(ccsd_t_tot, qcdb.variable('current energy'), tnm() + ' Current', atol=atol)
+    assert compare_values(ccsd_t_tot, qcdb.variable('ccsd(t) total energy'), tnm() + ' CCSD(T)', atol=atol)
+                                                                                               
+    assert compare_values(ccsd_t_corl, qcdb.variable('current correlation energy'), tnm() + ' CCSD(T)', atol=atol)
+    assert compare_values(ccsd_t_corl, qcdb.variable('ccsd(t) correlation energy'), tnm() + ' CCSD(T)', atol=atol)
+
+    assert compare_values(ccsd_tot, qcdb.variable('ccsd total energy'), tnm() + ' CCSD', atol=atol)
+    assert compare_values(ccsd_corl, qcdb.variable('ccsd correlation energy'), tnm() + ' CCSD', atol=atol)
+
+    assert compare_values(t_corl, qcdb.variable('(t) correction energy'), tnm() + ' (T)', atol=atol)
```

Comment:
btw, the 3rd/"label" argument is mostly good for debugging and unlike earlier psi4 days never gets seen under normal operations. so don't have to be elaborate and can leave blank.